### DISCUSSION TITLE
Issue with layer and route_width in route_bundle()

### DISCUSSION THREAD

**Discussion by aj-adkins** [gdsfactory/gdsfactory#3411] [General]
If I have some arrangement of objects I want to connect with a bundle like this,

```wg1 = top << gf.components.straight()
wg2 = top << gf.components.straight()
wg2.dmovey(50)
wg3 = top << gf.components.straight()
wg3.dmove((50, 20))
wg4 = top << gf.components.straight()
wg4.dmove((50, 70))

gf.routing.route_bundle(top, [wg1['o2'], wg2['o2']], [wg3['o1'], wg4['o1']])
```
then I get the following error:
```
ValueError: Either cross_section=None or layer=None and route_width=None must be provided
```

If I add the layer and route_width as the error requests, 

```
gf.routing.route_bundle(top, [wg1['o2'], wg2['o2']], [wg3['o1'], wg4['o1']], layer=(1, 0), route_width=0.5)
```
then I get this error:
```
ValueError: Cannot have both layer=(1, 0) and cross_section=functools.partial(<function cross_section at 0x000001EDE5FE79C0>, layer=(1, 0), width=0.5) provided. Choose one.
```
How can I use the route_bundle function without using the cross section?

**Comment by joamatab** [comment]
Hi AJ!

Thank you for reporting this

See fix here
https://github.com/gdsfactory/gdsfactory/pull/3413

**Reply by aj-adkins** [reply]
Hi Joaquin, thanks so much for the quick update!

**Reply by joamatab** [reply]
Thank you for reporting it :)

You can upgrade by running:
```
pip install gdsfactory --upgrade
```
That should resolve the issue. Let me know if you encounter any further problems!
